modules/apps/dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/resources/META-INF/resources/template_selector/page.jsp

```diff
@@ -26,14 +26,15 @@ Group ddmTemplateGroup = GroupLocalServiceUtil.getGroup(ddmTemplateGroupId);
 %>
 
 <clay:content-row
+	floatElements=""
```

Comment:
Just to follow the pattern we are using in DXP I'd add the value `"true"` or `"<%= true %>"`

<img width="1002" alt="Screenshot 2020-08-11 at 18 05 26" src="https://user-images.githubusercontent.com/46778114/89921041-7d15a200-dbfd-11ea-9bb4-c342ecda2a48.png">

Reply:
`<%= true %>` does not compile, and `"true"` is not correct, the result class by using this is:
`autofit-float-true`

if you take a look at:
`liferay-portal/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ContentRowTag.java`
line: `84`
what we need is: `if (_floatElements.equals(StringPool.BLANK)) {`

So we need `floatElements=""`and probably change all `floatElements="true"` to `floatElements=""`

Reply:
🤔 I think we should change the API to mirror the React one https://github.com/liferay/clay/blob/master/packages/clay-layout/src/Content.tsx#L65

Reply:
Whatever it takes to fix it, I think we have some autofit-float currently not working properly, I'm working on validating and create an issue if needed